CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# Code Ownership for buzzmobile.
+# Order matters. The last match is used, so more specific owners should be 
+# last.
+
+# Default owners/reviewers for anything.
+* @irapha @joshuamorton
+
+# Language specific reviewers:
+*.py @iraphael @joshuamorton
+*.cpp @chsahit
```

Comment:
As far as I know, since you have these 3 people listed under `*`, this dosen't make a difference, since those people will be added as reviews regardless of whether it changes a `py` or `cpp` or neither.

[docs](https://github.com/blog/2392-introducing-code-owners)